TSPL.docc/ReferenceManual/Types.md

```diff
@@ -948,6 +949,45 @@ https://github.com/apple/swift-evolution/blob/main/proposals/0335-existential-an
 >
 > *boxed-protocol-type* → **`any`** *type*
 
+## Type-Parameter Pack
+
+XXX OUTLINE:
+
+- `each T` creates a type-parameter pack
```

Comment:
I wonder if we actually want the hyphen in `type-parameter pack`? Looking at [existing documentation](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/genericparametersandarguments/) for generic type parameters, I do not see the hyphen used.